reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md

```diff
@@ -44,9 +44,7 @@ Name    Enabled  Description
 User02  True     Description of this account.
 ```
 
-This command creates a local user account.
-The command does not specify the *AccountExpires* parameter.
-Therefore, the account does not expire.
+This command creates a local user account without an initial password.  This is not a recommended "best practice".
```

Comment:
I believe we could add this in as well but is there a reason you removed the other information?

Reply:
It didn't seem relevant to point out one unused parameter when using the default (most-basic) example.  No worries if you prefer to leave that part in.